tests/api/runtimes/test_spark.py

```diff
@@ -252,3 +253,55 @@ def test_cores(
 
         self.execute_function(runtime)
         self._assert_custom_object_creation_config(expected_cores=expected_cores)
+
+    @pytest.mark.parametrize(
+        "mount_v3io_to_executor",
+        [True, False],
+    )
+    def test_with_igz_spark(
```

Comment:
Since you're only testing the mounts here, please change the name of this test to something like `test_with_igz_spark_mounts_creation`.
Another option is to modify the test to also check that `with_igz_spark` does what it's expected to do, including the jars and history server config...